game-core/src/test/java/games/strategy/engine/xml/TestGameLoader.java

```diff
@@ -17,6 +17,10 @@
 
 /**
  * Fake implementation of {@link IGameLoader} that does nothing.
+ *
+ * <p>
+ * Note that instances of this class are typically created via reflection. See the various game XML test resources.
```

Comment:
Interested to know a bit more about this, I thought that we eliminated all reflection regarding game loading already.

Reply:
Unfortunately not. :disappointed:  `GameParser` still instantiates `IGameLoader` via reflection.

https://github.com/triplea-game/triplea/blob/38802ae8d4f9293f1af4accc414f117414b0d679/game-core/src/main/java/games/strategy/engine/data/GameParser.java#L518-L525

https://github.com/triplea-game/triplea/blob/38802ae8d4f9293f1af4accc414f117414b0d679/game-core/src/main/java/games/strategy/engine/data/GameParser.java#L452-L459